src/vmm/src/device_manager/persist.rs

```diff
@@ -193,6 +214,16 @@ impl DeviceStates {
 
         Ok(())
     }
+
+    fn entropy_serialize(&mut self, target_version: u16) -> VersionizeResult<()> {
+        if target_version < 4 && self.entropy_device.is_some() {
```

Comment:
If there some constant we can refer to for this magic number?

Reply:
It doesn't look like. The number is just +1 of what was the previous maximum. The whole versionizing thingy needs a better design I think.

Reply:
Ah, yeah for some reason I thought it mapped to one of the snapshot versions. Forgot we had separate versions for each struct. Maybe those should've been tied to  each other, but that ship has long sailed I guess.